code/modules/projectiles/projectile/energy.dm

```diff
@@ -147,3 +147,63 @@
 	armour_penetration = 10 // It can have a little armor pen, as a treat. Bigger than it looks, energy armor is often low.
 	shield_buster = TRUE
 	is_reflectable = FALSE //I will let eswords block it like a normal projectile, but it's not getting reflected, and eshields will take the hit hard.
+
+/obj/item/projectile/energy/detective
+	name = "placeholder name"
+	icon_state = "omnilaser"
+	light_color = LIGHT_COLOR_CYAN
+	damage = 5
+	stamina = 25
+	eyeblur = 5
+
+/obj/item/projectile/energy/detective/overcharged
+	name = "overcharged shot"
+	icon_state = "spark"
+	light_color = LIGHT_COLOR_DARKRED
+	damage = 45
+	stamina = 15
+	eyeblur = 10
```

Comment:
```suggestion
	eyeblur = 20 SECONDS
```

Reply:
both of these are assuming you meant these to be in cycles. if you meant 10 deciseconds, then please do use the `SECONDS` define. 10/20 seconds of eyeblur is quite a while.